projects/angular-sdk/src/lib/components/user-profile/user-profile.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, ElementRef, Input, OnChanges, OnInit } from '@angular/core';
+import DescopeUserProfileWidget from '@descope/user-profile-widget';
+import { ILogger } from '@descope/web-component';
+import { DescopeAuthConfig } from '../../types/types';
+
+@Component({
+	selector: 'user-profile',
+	standalone: true,
+	template: ''
+})
+export class UserProfileComponent implements OnInit, OnChanges {
+	projectId: string;
+	baseUrl?: string;
+	@Input() widgetId: string;
+
+	@Input() theme: 'light' | 'dark' | 'os';
+	@Input() debug: boolean;
+	@Input() logger: ILogger;
+
+	private readonly webComponent = new DescopeUserProfileWidget();
+
+	constructor(
+		private elementRef: ElementRef,
+		descopeConfig: DescopeAuthConfig
+	) {
+		this.projectId = descopeConfig.projectId;
+		this.baseUrl = descopeConfig.baseUrl;
+	}
+
+	ngOnInit() {
+		this.setupWebComponent();
+		this.elementRef.nativeElement.appendChild(this.webComponent);
+	}
+
+	ngOnChanges(): void {
+		this.setupWebComponent();
+	}
+
+	private setupWebComponent() {
+		this.webComponent.setAttribute('project-id', this.projectId);
```

Comment:
Don't we need to have here the logout callback?

Reply:
done